models/_sector/cex/flows/cex_flows.sql

```diff
@@ -0,0 +1,63 @@
+{{ config(
+        
+        schema='cex',
+        alias = 'flows',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['blockchain', 'flow_type', 'unique_key'],
```

Comment:
can you add the new incremental predicate property in the config block? filtering target to same timeframe as source has been beneficial on performance:
https://github.com/duneanalytics/spellbook/blob/main/models/_sector/dex/trades/dex_trades.sql#L9

this would apply to all incremental models moving forward as a best practice